7.3.Частые_сценарии/7.3.7.py

```diff
@@ -0,0 +1,9 @@
+# Асимптотическое приближение
+
+from math import *
```

Comment:
Пишу этот замечаний каждому - используйте только те функции и константы из модуля, которые нужны:
```suggestion
from math import log
```

Reply:
Импортируя все функции, больше памяти используется